documentation/configuration/basic.md

```diff
@@ -43,6 +43,10 @@ If this takes longer than 10 minutes, you may have an issue with your Home Assis
 
 _If something failed, you will be taken back to the [Add your configuration step](#add-your-configuration)_
 
+:::info
+The initial startup can take up to 1 hour, for more details click [here](https://hacs.xyz/docs/faq/initial_startup)

```

Comment:
The 1 hour will change, I don't want to change this information multiple places in the future.

```suggestion
[Click here](https://hacs.xyz/docs/faq/initial_startup) for more information about the first startup of HACS.

```